PocketKit/Sources/SaveToPocketKit/SavedItem/SavedItemViewModel.swift

```diff
@@ -65,6 +65,7 @@ class SavedItemViewModel {
 
         for item in extensionItems {
             guard let url = try? await url(from: item) else {
+                tracker.track(event: Events.SaveTo.unableToSave())
```

Comment:
@dskuza should we pass in the data we have?

Reply:
I guess we could throw with some of the data that couldn't be processed, but that seems a little overkill for now. Maybe something to revisit.